### PR TITLE
docs: add jsdoc to lending sdk

### PR DESCRIPTION
**Pull request by LuPoYi**

### CODE REVIEW

File: packages/lending/src/adapter.ts [hunk 1]
```diff
@@ -590,15 +687,35 @@ export class Adapter extends common.Web3Toolkit {
...
-  // 2. swap destToken to srcToken
-  // 3. repay srcToken
-  // 4. add fund aDestToken
-  // 5. withdraw destToken
-  // 6. flashloan repay destToken
-  // @param srcToken Borrowed token, repaid token
-  // @param destToken Deposit token, collateral token
-  async deleverage({ account, portfolio, srcToken, srcAmount, destToken, slippage = defaultSlippage }: OperationInput) {
+  /**
+   * Leverage long enables user to reduce the collateral exposure in a single step using a flash loan to
```
Comment (hihiben) [inline packages/lending/src/adapter.ts]: Incorrect explanation.

**Review by chouandy — COMMENTED** [review]
Would it be better to use `params` instead of `param` for `@param {OperationInput} param`?

File: packages/lending/src/adapter.ts (hunk 1 above)
Comment (LuPoYi) [reply packages/lending/src/adapter.ts]: fixed.

**Review by hihiben — APPROVED** [review]
LGTM

**Review by chouandy — APPROVED** [review]
LGTM

**Pull request merged**